### DISCUSSION TITLE
ControlNet v1.1.445 in ForgeUI?

### DISCUSSION THREAD

**Discussion by blodytrav3l3r** [lllyasviel/stable-diffusion-webui-forge#695] [General]
Can I use controlnet from https://github.com/Mikubill/sd-webui-controlnet in ForgeUI
https://github.com/lllyasviel/stable-diffusion-webui-forge

**Comment by gohan2091** [comment]
I would like to know how to get an updated version of Control Net in Forge.

**Comment by strawberrymelonpanda** [comment]
1) @blodytrav3l3r No, you can't.  Forge disables it and uses its own version.
2) @gohan2091 You don't. 😢 

> huchenlei
> Apr 20, 2024
> Collaborator Author

> Unfortunately lllyasviel did not give us collaborators merge permission in forge repo, so any work need to wait until he comes back.

Maybe someone will make a workaround, but none currently exist that I know of.

**Comment by gohan2091** [comment]
Has Forge been abandoned then? There has been no updates and important features are not added :(

**Comment by altoiddealer** [comment]
If its any consolation, I added a simple PR for Forge ControlNet - making API usage...useful.
Don't expect it to ever be merged or anything
